Readme.md

```diff
@@ -0,0 +1,494 @@
+[1]: https://www.mongodb.com/atlas/database
+
+[2]: https://www.mongodb.com/docs/atlas/sample-data/
+
+[3]: https://www.mongodb.com/cloud/atlas/register
+
+[4]: https://www.mongodb.com/docs/drivers/kotlin/coroutine/current/
+
+[5]: https://github.com/mongodb-developer/kotlin-driver-quick-start
+
+# Getting started with MongoDB for building backend Applications using the official Kotlin driver
```

Comment:
Since this is a series, it might be worth adding a "Part 1" to the title or being more explicit that this tutorial covers basic CRUD with Kotlin. This way users know what they are signing up for when they click the link.

Reply:
As discussed earlier, this is a standalone article.